Isosurfacing_3/examples/Isosurfacing_3/marching_cubes_inrimage.cpp

```diff
@@ -0,0 +1,52 @@
+#include <CGAL/Simple_cartesian.h>
+
+#include <CGAL/Isosurfacing_3/Cartesian_grid_3.h>
+#include <CGAL/Isosurfacing_3/Explicit_Cartesian_grid_domain_3.h>
+#include <CGAL/Isosurfacing_3/marching_cubes_3.h>
+
+#include <CGAL/boost/graph/IO/OFF.h>
+
+#include <vector>
+
+using Kernel = CGAL::Simple_cartesian<double>;
+using Point = typename Kernel::Point_3;
+using Grid = CGAL::Isosurfacing::Cartesian_grid_3<Kernel>;
+
+using Point_range = std::vector<Point>;
+using Polygon_range = std::vector<std::vector<std::size_t> >;
+
+int main(int, char**)
+{
+  const std::string fname = "../examples/Isosurfacing_3/FullHead.inr";//CGAL::data_file_path("images/skull_2.9.inr");
```

Comment:
Anybody of you has this file?  Is it more challenging than skull?

Reply:
Yes, it's a lot more challenging. The file is attached here:
https://github.com/CGAL/cgal/pull/6849#issuecomment-1434349696
But I'm not sure about the rights we have on the data, so I've only used it to test the implementation for myself.
I just forgot to remove it before committing.